sklearn/tests/test_pipeline.py

```diff
@@ -1526,6 +1526,48 @@ def fit(self, X, y):
     check_is_fitted(pipeline)
 
 
+def test_feature_union_check_if_fitted():
+    class Estimator(BaseEstimator):
```

Comment:
We can use `from sklearn.utils._testing import MinimalTransformer` for a minimal transformer.